src/components/addBookForm.js

```diff
@@ -1,20 +1,42 @@
-/* eslint-disable react/jsx-props-no-spreading, no-console */
-import React from 'react';
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
 import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import { v4 as bookId } from 'uuid';
+import { addBook } from '../redux/books/books';
 import './addBookForm.css';
 
 export default function AddBookForm() {
   const {
     register, handleSubmit, formState: { errors },
   } = useForm();
 
-  const onSubmit = (data) => console.log(data);
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [category, setCategory] = useState('');
+
+  const handleCategoryChange = (e) => {
+    setCategory(e.target.value);
+  };
+
+  const handleTitleChange = (e) => {
+    setTitle(e.target.value);
```

Comment:
- [ OPTIONAL] Please it will be nice to clear the inputs after adding a book. After adding book, the values still remain on the inputs and you can add them several times, please kindly fix that.